src/api/eth/getGasStationPrice.js

```diff
@@ -1,12 +1,11 @@
-const { get } = require('request-promise')
-
+const get = require('../../lib/dvf/get-generic')
 /**
  * Provides a safe average gas price
  */
 
 module.exports = async (dvf)  => {
   try {
-    const res = await get(`${dvf.config.gasApi}/json/ethgasAPI.json?api-key=${dvf.config.gasStationApiKey || ''}`)
+    const res = await get(dvf, `${dvf.config.gasApi}/json/ethgasAPI.json?api-key=${dvf.config.gasStationApiKey || ''}`)
```

Comment:
given the previous comments, we cannot use get-generic here as it prepend `dvf.config.api`  and we need `dvf.config.gasApi` here, so let's switch to plain `request.get`  from `@rhino.fi/dvf-utils`

Reply:
Do we need to parse the response in the following line then? As it looks like we don't, but I'm not sure

`dvf.config.defaultGasPrice = parseInt((JSON.parse(res).average * 1.25 *100000000))`

Reply:
I guess not, If you remove the JSON.parse I can test to confirm